src/widgets/window.js

```diff
@@ -258,7 +259,7 @@ var FlatsealWindow = GObject.registerClass({
             showPermissionsButton = false;
         }
 
-        const isFolded = this._headerLeaflet.folded;
+        const isFolded = this._contentLeaflet.folded;
         if (isFolded) {
             showApplicationsButton = true;
             showPermissionsButton = true;
```

Comment:
Minor issue:

Reply:
I think you may have forgotten to attach an image?

Reply:
My bad I was trying to use peek to record something and accidently clicked on "single comment" :laughing: